<launch>
    <!-- <arg name="radius_comm" value="6.9284" /> -->
    <arg name="radius_comm" value="8.0" />

<!--    <include file="$(find map_generator)/launch/map_generator.launch">-->
<!--        <arg name="map_size_x_" value="$(arg map_size_x)"/>-->
<!--        <arg name="map_size_y_" value="$(arg map_size_y)"/>-->
<!--        <arg name="map_size_z_" value="$(arg map_size_z)"/>-->
<!--        &lt;!&ndash; Number of circle and pillar obstacles &ndash;&gt;-->
<!--        <arg name="c_num" value="0"/>-->
<!--        <arg name="p_num" value="10"/>-->
<!--    </include>-->

<!--    <node pkg="optimizer" name="test_los" type="test_los" output="screen">-->

<!--        <param name="los_checker/x_size" value="$(arg map_size_x)" type="double"/>-->
<!--        <param name="los_checker/y_size" value="$(arg map_size_y)" type="double"/>-->
<!--        <param name="los_checker/z_size" value="$(arg map_size_z)" type="double"/>-->
<!--        <param name="grid_map/resolution" value="1.0" type="double"/>-->
<!--        <param name="grid_map/radius2res" value="4.0" type="double"/>-->

<!--        <remap from="/global_map" to="/map_generator/global_cloud"/>-->
<!--    </node>-->

    <node pkg="optimizer" name="test_pruning" type="test_pruning" output="screen">

    </node>

</launch>